mdal/api/mdal.h

```diff
@@ -397,6 +413,8 @@ MDAL_EXPORT int MDAL_M_datasetGroupCount( MDAL_MeshH mesh );
 
 /**
  * Returns dataset group handle
+ *
+ * Since 0.9.0 - if the driver has Write Capability for this DatasetGroup, it is opened in edit mode
```

Comment:
I strongly suggest to not change the behaviour of the exisisting functions. they are used all over the place in qgis and we would need to make sure to close edit mode? Also if you want to only read the dataset you cannot do it. Can you create a new function to start edit mode of existing mesh dataset group instead?

Reply:
Ok - I removed these changes from this PR and will add something else later

Reply:
can you also please change the api doc for this function accordingly?

Reply:
@runette please change the api doc for this function accordingly? and i will merge afterwards

Reply:
API Doc corrected - good catch